packages/components/icon/src/Icon.tsx

```diff
@@ -39,3 +39,5 @@ export function Icon({
     </>
   )
 }
+
+Icon.displayName = 'Icon'
```

Comment:
we hadnt made any displayName to a component yet till this one

Reply:
Yes! I discussed this with @Powerplex this yesterday. Even though most of the times is not necessary since is inferred from the function name (see https://legacy.reactjs.org/docs/react-component.html#displayname) we noticed that is needed by storybook canvas. I will created another task to set it in all the components and add it to the generator if you all agree